src/utils/permissions.ts

```diff
@@ -0,0 +1,321 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
Nitpick: this doesn't need to be disabled for the whole file, just lines 117-122